tensorflow/contrib/framework/python/ops/variables_test.py

```diff
@@ -810,5 +810,27 @@ def testInitFromCheckpointWithScopes(self):
       self.assertAllEqual(init_value0, var0.eval())
       self.assertAllEqual(init_value1, var1.eval())
 
+class ZeroInitializerOpTest(tf.test.TestCase):
+
+  def _testZeroInitializer(self, shape, dtype):
+    var0 = tf.Variable(tf.zeros(shape, dtype=dtype))
+    var1 = tf.Variable(tf.ones(shape, dtype=dtype))
+    var0_zero = tf.contrib.framework.zero_initializer(var0)
+    var1_zero = tf.contrib.framework.zero_initializer(var1)
```

Comment:
yes, I'll add memory test.

发自我的 iPhone

> 在 2016年9月13日，上午1:16，Alexandre Passos <notifications@github.com> 写道：
> 
> In tensorflow/contrib/framework/python/ops/variables_test.py:
> 
> > @@ -810,5 +810,27 @@ def testInitFromCheckpointWithScopes(self):
> >        self.assertAllEqual(init_value0, var0.eval())
> >        self.assertAllEqual(init_value1, var1.eval())
> >  
> > +class ZeroInitializerOpTest(tf.test.TestCase):
> > +
> > +  def _testZeroInitializer(self, shape, dtype):
> > +    var0 = tf.Variable(tf.zeros(shape, dtype=dtype))
> > +    var1 = tf.Variable(tf.ones(shape, dtype=dtype))
> > +    var0_zero = tf.contrib.framework.zero_initializer(var0)
> > +    var1_zero = tf.contrib.framework.zero_initializer(var1)
> So how about we add the unit test I suggested, checking memory usage, and then see whether we can make that syntax work?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>